pkg/k8s/watchers/endpointsynchronizer.go

```diff
@@ -176,12 +184,14 @@ func (epSync *EndpointSynchronizer) RunK8sCiliumEndpointSync(e *endpoint.Endpoin
 							return err
 						}
 
+						scopedLog.WithField("cep_uid", localCEP.UID).Debug("storing CEP UID after create")
```

Comment:
Let's define `cepGUID` (nit on underscores just to conform to existing camel-case) as a logfield inside the logfields package and use it throughout. It seems useful enough now